CHANGELOG.md

```diff
@@ -10,6 +10,13 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 [Unreleased][Unreleased_log]
 --------------
 
+## Breaking Changes
+- liboembed is renamed to liboembedtls.
```

Comment:
An install of OE SDK built from current master has the following mbed libraries:
- libmbedcrypto.a 
- libmbedtls.a     
- liboecryptombed.a 
- libmbedx509.a   

liboembed is not in the list. Did you mean libmbedtls.a ?

It is not clear what the difference is between liboecryptombed.a and libmbedcrypto.a.

Reply:
There is a typo, thanks for catching this.